python/ray/data/dataset.py

```diff
@@ -2535,6 +2535,11 @@ def write_json(
             >>> ds = ray.data.range(100)
             >>> ds.write_json("local:///tmp/data")
 
+            Write the dataset as JSONL files to a local directory.
```

Comment:
I'm not sure if we need both of these examples. They both write to JSON format files.

Reply:
I think we can keep only one example.